cdap-security/src/test/java/co/cask/cdap/security/authorization/DummyAuthEnforce.java

```diff
@@ -45,6 +45,14 @@ public void testSingleAction(@Name("namespaceId") NamespaceId namespaceId) throw
       throw new EnforceNotCalledException();
     }
 
+    @Deprecated // NOTE : Not deprecated, tests that the presence of other annotations does not affect class rewrite
```

Comment:
can we use something like `@VisibleForTesting`, then?

Reply:
yeah, thats a better annotation to use for testing. i have updated it.